agent/src/trident.rs

```diff
@@ -284,6 +284,7 @@ impl Trident {
 
         if running_in_container() && config.kubernetes_cluster_id.is_empty() {
             config.kubernetes_cluster_id = Config::get_k8s_cluster_id(&session);
+            warn!("Please confirm that k8s is used to limit cpu and memory");
```

Comment:
When running in a K8s pod, the cpu and memory limits notified by deepflow-server will be ignored, please make sure to use K8s for resource limits.

Reply:
When running in a K8s pod, the cpu and memory limits declared by deepflow-server will be ignored, please make sure to use K8s for resource limits